content/applications/sales/subscriptions/ecommerce.rst

```diff
@@ -19,33 +20,38 @@ recurrence period.
 Create recurrence periods as product variants
 =============================================
 
-To set up each recurrence period as a product variant, go to :menuselection:`Subscriptions -->
-Subscriptions --> Products` and select a product. In the :guilabel:`Attributes & Variants` tab,
-click :guilabel:`Add a line`.
+To set up each recurrence period as a product variant, go to :menuselection:`Subscriptions app -->
+Products`, and select a product. In the :guilabel:`Attributes & Variants` tab, click :guilabel:`Add
+a line`.
 
-Create an :guilabel:`Attribute` called `Billing Period` (or something similar) by typing in the
-name and clicking :guilabel:`Create`. This attribute name appears as the option heading on the
-product page of the eCommerce shop.
+Then proceed to create an :guilabel:`Attribute` called `Billing Period` (or something similar), by
+typing in the name, and clicking :guilabel:`Create` from the mini drop-down menu that appears. This
+attribute name appears as option heading on the product page of the eCommerce shop.
 
 Next, create :guilabel:`Values` that correspond to the recurrence periods that are configured in
-the :guilabel:`Time-based pricing` tab of the product form. Type in the name of the recurrence
-period, then click :guilabel:`Create`. These value names appear as selectable options on the
-product page of the eCommerce shop.
+the :guilabel:`Recurring Prices` tab of the product form.
+
+To do that, type in the name of the recurrence period in the :guilabel:`Values` column, on the same
+:guilabel:`Attribute` line, in the :guilabel:`Attributes & Variants` tab. Then, click
+:guilabel:`Create` from the mini drop-down menu that appears.
+
+These value names appear as selectable options on the product page of the eCommerce shop.
 
 .. image:: ecommerce/recurrence-period-attributes-variants.png
    :align: center
    :alt: Recurrence periods configured as product variants in the "Attributes & Variants" tab of
          the product form.
 
-Click on the :guilabel:`☁️ (cloud)` icon at the top of the page to save manually. After saving, a
-:guilabel:`Product Variants` column appears on the :guilabel:`Time-based pricing` tab. Assign the
-product variants to their corresponding recurrence periods and prices.
+With those configurations in place and saved, a :guilabel:`Product Variants` column appears in the
+:guilabel:`Recurring Prices` tab. Proceed to assign the different :guilabel:`Product Variants` to
+their corresponding recurrence :guilabel:`Period` and :guilabel:`Price`.
 
 .. image:: ecommerce/product-variants-time-based-pricing.png
    :align: center
    :alt: Product variants on the "Time-based pricing" tab of the product form.
 
-The product variants are now available for selection on the eCommerce product page.
+After following those aforementioned steps, the product variants are available for selection on the
+eCommerce product page.
```

Comment:
it might be helpful to include menuselection steps to get to the eCommerce product page?

Reply:
This would be a difficult task, as they can implement the lesson from this doc on *any* subscription product, which could be on *any* page, of their online/eCommerce store. So, I think this more generic allusion to where it would be located is the best/most apt approach for this particular circumstance.